result/brace-style.js

```diff
@@ -0,0 +1,209 @@
+if (foo) {
```

Comment:
__[development][eslint]__
* Error - Opening curly brace appears on the same line as controlling statement. (brace-style)